README.md

```diff
@@ -1,50 +1,56 @@
+<div align="center">
+  <img src="./apps/wing/logo/banner.gif" alt="Wing Banner" height="360px">
+</div>
+
 <h1 align="center">Welcome to the Wing Language! :wave:</h1>
 <p align="center">
-  <a href="https://www.winglang.io/learn/">Take a Tour</a>
-  ▪︎
-  <a href="https://www.winglang.io/docs/">Getting Started</a>
-  ▪︎
-  <a href="http://t.winglang.io/slack">Join Slack</a>
-  ▪︎
-  <a href="https://www.winglang.io/docs/category/faq">FAQ</a>
-  ▪︎
-  <a href="https://www.winglang.io/contributing/status#roadmap">Roadmap</a>
-  ▪︎
-  <a href="https://github.com/winglang/wing/issues">Issues</a>
-  ▪︎
-  <a href="https://github.com/winglang/wing/discussions">Discussions</a>
-  ▪︎
-  <a href="https://www.winglang.io/contributing/">Contribute</a>
+
+<a href="https://www.winglang.io/learn/">Take a Tour</a>
+▪︎
+<a href="https://www.winglang.io/docs/">Getting Started</a>
+▪︎
+<a href="http://t.winglang.io/slack">Join Slack</a>
+▪︎
+<a href="https://www.winglang.io/docs/category/faq">FAQ</a>
+▪︎
+<a href="https://www.winglang.io/contributing/status#roadmap">Roadmap</a>
+▪︎
+<a href="https://github.com/winglang/wing/issues">Issues</a>
+▪︎
+<a href="https://github.com/winglang/wing/discussions">Discussions</a>
+▪︎
+<a href="https://www.winglang.io/contributing/">Contribute</a>
+
```

Comment:
I'm not sure why but it looks like this broke the centering so now these are left-aligned. Was that intentional?

Reply:
<img width="1314" alt="image" src="https://github.com/winglang/wing/assets/1237390/8632b36d-34bd-4e6b-abaf-8bd42efc283b">

Reply:
Hmm, the preview in VS code didn't look weird. I'll check it out